### PR TITLE
Add unwrap keyword to radius of gyration

### LINKED ISSUE #1760: Radius of Gyration doesn't work with periodic boundaries.
### Expected behaviour

Calculating correct radius of gyration when protein is split on the periodic boundary and we set `pbc=True`

### Actual behaviour

Neither with the pbc option set or without it do I get the correct radius of gyration. This is because to correctly handle the split of a protein on periodic boundaries we would need to recombine proteins that are split (unwrap them)

[Example protein](https://gist.github.com/kain88-de/cca3fe830772931d76ddca24273dfd53) for which the correct value should be 2-4 nm.

### Code to reproduce the behaviour

``` python
import MDAnalysis as mda

u = mda.Universe('protein.gro')
u.atoms.radius_of_gyration(pbc=True)

```

### Currently version of MDAnalysis:
(run `python -c "import MDAnalysis as mda; print(mda.__version__)"`)

### PR DESCRIPTION
**Pull request by NinadBhat**
Fixes #1760 

Changes made in this Pull Request:
 - Adds unwrap keyword to radius of gyration

PR Checklist
------------
 - [x] Tests?
 - [x] Docs?
 - [ ] CHANGELOG updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

**Comment by NinadBhat** [comment]
Will rebase after #2290 and #2288

File: package/MDAnalysis/core/topologyattrs.py [hunk 1]
```diff
@@ -953,15 +953,23 @@ def radius_of_gyration(group, **kwargs):
 
 
         .. versionchanged:: 0.8 Added *pbc* keyword
+        .. versionchanged:: 0.20.0 Added *unwrap* parameter
 
         """
         atomgroup = group.atoms
         pbc = kwargs.pop('pbc', flags['use_pbc'])
         masses = atomgroup.masses
+        unwrap = kwargs.pop('unwrap', False)
```
Comment (richardjgowers) [inline package/MDAnalysis/core/topologyattrs.py]: You need to update the docstring to include all keywords

**Comment by NinadBhat** [comment]
Not sure why this PR is showing changes from #1991 even after rebase. Should I create a new PR?

**Comment by NinadBhat** [comment]
Copy of this PR is at #2299

**Pull request closed without merging**